tests/compat.py

```diff
@@ -1,4 +0,0 @@
-try:
```

Comment:
Removing this ``compat`` module is OK, but you can import ``mock`` just because ``pytest-mock`` requires the ``mock`` package that adds ``mock`` functionality to ``python2.7``.

I'm going to double check all tests where you're using ``import mock`` because it's better to use the ``mocker`` fixture instead of ``mock`` (you have installed ``pytest-mock`` for that).

Why bother using ``mocker`` instead of ``mock``? Check [pytest-mock][1] documentation ("*Why bother with a plugin?*" section)

[1]: https://pypi.python.org/pypi/pytest-mock/1.2